Gruntfile.js

```diff
@@ -269,6 +282,16 @@ module.exports = function(grunt) {
                       'concat',
                       'uglify']);
 
+  grunt.registerTask('update-webdriver', function () {
```

Comment:
Is there a way to break this process out into it's own module? Seems like the same function will be added to most of the grunts.